### ISSUE TITLE
Is this software available via macos brew package manager?

### ISSUE THREAD

**Issue by marhar** [G2-Games/sixaxis-pair_rs#6]
Are there any plans for this?  I'm teaching a class, and the students are probably not to the level of downloading and building this.

**Comment by G2-Games** [comment]
No, currently it is not. Do you know what it would take to put it on there?

**Comment by marhar** [comment]
I'm no brewmaster, but my understanding is that the package is added in the Formula directory of:

https://github.com/Homebrew/homebrew-core

following these instructions:

https://github.com/mapbox/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md

**Comment by G2-Games** [comment]
Ok. I have time to look into this later this week.

FYI, if your students are able to install `cargo` (the rust package manager), they can run:

```bash
cargo install --git https://github.com/G2-Games/sixaxis-pair_rs
```

And cargo will take care of cloning, building, and installing. There should be no additional dependencies with this package, so it should (maybe, I haven't tested that many) work on any system.
